addon/mixins/axes.js

```diff
@@ -16,11 +16,61 @@ export default Ember.Mixin.create({
   graphicHeight: null,
   minXTicks: 3,
   minYTicks: 3,
-  tickSpacing: 50,
-
   minAxisValue: 0,
   maxAxisValue: 0,
 
+  /**
+   * We used to have only one option to set tick spacing for both x and y axes.
+   * We keep this attribute for backward compatibility.
+   * @type {number}
+   * @deprecated This will be deprecated in version 1.0.
+   */
+  tickSpacing: 50,
+
+  /**
+   * Tick spacing on X axis. Set this value if we want a different tickSpacing
+   * for X axis other than the default one set in tickSpacing for both axes
+   * @type {number}
+   */
+  tickSpacingX: null,
+
+  /**
+   * Tick spacing on Y axis. Set this value if we want a different tickSpacing
+   * for Y axis other than the default one set in tickSpacing for both axes
+   * @type {number}
+   */
+  tickSpacingY: null,
+
+  /**
+   * This will be used for all internal calculation of tick spacing on X axis.
+   * We set higher priority if the specific tickSpacingX's value is set.
+   * @type {number}
+   * @private
+   */
+  _innerTickSpacingX: Ember.computed('tickSpacingX', 'tickSpacing',
+    function() {
```

Comment:
The last argument to `Ember.computed` after the dependent keys should be a string?